templates/rustdoc/topbar.html

```diff
@@ -62,10 +62,7 @@
                             {%- if krate.github_metadata -%}
                                 <li class="pure-menu-item">
                                     <a href="{{ krate.repository_url }}" class="pure-menu-link">
-                                        {{ "github" | fab(fw=true) }}
-                                        {{ "star" | fas(fw=true) }} {{ krate.github_metadata.stars }}
-                                        {{ "code-branch" | fas(fw=true) }} {{ krate.github_metadata.forks }}
-                                        {{ "exclamation-circle" | fas(fw=true) }} {{ krate.github_metadata.issues }}
+                                        {{ "code-branch" | fas(fw=true) }} Repository
```

Comment:
Now that it's more clear I would still show the GitHub metadata.

Reply:
And I answer the same: it looks kinda weird in the menu which is already quite "heavy". Just having "repository" looks better in my opinion. If people want more information, the docs.rs crate page is a much better place.

Reply:
(I added a screenshot with the current menu look to show what it looks like 😃)